### ISSUE TITLE
How to learn on other datasets

### ISSUE THREAD

**Issue by W-hary** [facebookresearch/CutLER#42]
Thanks for the author's contribution
We noticed that the author mentioned in Features that CutLER can learn unsupervised object detectors and instance segmentors solely on ImageNet-1K.
Whether CutLER can be used for our datasets, which is not included in the ImageNet-1K？

**Comment by xiaoxin05** [comment]
hello，i have some problems ，can you help me ？

**Comment by frank-xwang** [comment]
Hi, CutLER can be used for custom datasets! You can follow the pipeline of training CutLER on ImageNet-1k:
1. Cut: Use MaskCut to provide segmentation masks for multiple instances of each image, following [this section](https://github.com/facebookresearch/CutLER/blob/main/README.md#generating-annotations-for-imagenet-1k-with-maskcut). And place the json file under "DETECTRON2_DATASETS/YOUR-DATASET/annotations/".
2. Learn: after getting all pseudo labels on your datasets, you can follow [this section](https://github.com/facebookresearch/CutLER/blob/main/README.md#unsupervised-model-learning) to start the unsupervised model learning process. Note: 1) If your dataset is very small, it is recommended to load our ImageNet pretrained model weights and fine-tune the model on your dataset. 2) you need to follow [detectron2](https://detectron2.readthedocs.io/en/latest/tutorials/datasets.html) to understand how to train a model on custom datasets. 
3. Self-training stage is optional. 

Hope it helps.

**Comment by frank-xwang** [comment]
Closing this issues. Please feel free to re-open it if you meet other issues.
